packages/cozy-harvest-lib/test/components/DeleteAccountButton.spec.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import { DeleteAccountButton } from 'components/DeleteAccountButton'
+
+const fixtures = {
+  account: {
+    _id: '61c683295560485db0f34b859197c581',
+    account_type: 'konnectest',
+    auth: {
+      username: 'foo',
+      passphrase: 'bar'
+    },
+    identifier: 'username'
+  }
+}
+
+const tMock = () => 'Mock button label'
+
+const props = {
+  t: tMock,
+  deleteAccount: jest.fn()
+}
+
+describe('DeleteAccountButton', () => {
+  beforeEach(() => {
+    jest.resetAllMocks()
+  })
+
+  it('should render without listeners', () => {
+    const component = shallow(
+      <DeleteAccountButton account={fixtures.account} {...props} />
+    ).getElement()
+    expect(component).toMatchSnapshot()
+  })
+
+  it('should run deleteAccount on click', () => {
+    const component = shallow(
+      <DeleteAccountButton account={fixtures.account} {...props} />
+    )
+    component.simulate('click')
+    expect(props.deleteAccount).toHaveBeenCalledWith(fixtures.account)
+    // the rest is asserted in the onSuccess
```

Comment:
I think you can remove this comment. It's difficult to know what "rest" is here.